src/components/Table/components/Row.tsx

```diff
@@ -93,8 +92,13 @@ function RowImpl<R extends Kinded, S>(props: RowProps<R>): ReactElement {
   const rowStyleCellCss = maybeApplyFunction(row as any, rowStyle?.cellCss);
   const levelIndent = style.levels && style.levels[level]?.rowIndent;
 
-  const rowCss = {
+  const containerCss = {
     ...Css.add("transition", "padding 0.5s ease-in-out").$,
+    ...(rs.isDraggedOver === DraggedOver.Above && Css.add("paddingTop", "35px").$),
+    ...(rs.isDraggedOver === DraggedOver.Below && Css.add("paddingBottom", "35px").$),
```

Comment:
**nit:** Truss has a lot of great shorthand ways or writing styles that hopefully make this sort of thing less verbose and easier. You could easily do these padding styles via: `Css.ptPx(35).$`.